docs/crashstorage/elasticsearch.rst

```diff
@@ -36,14 +36,14 @@ web app. Here is a list of those features:
   share a signature. Features include listing crash reports, listing user
   comments and showing aggregation on any field of the database.
 
-  Example: https://crash-stats.mozilla.com/signature/?signature=nsTimeout::~nsTimeout%28%29
```

Comment:
We can remove this line. Seems like unhelpful maintenance burden to have examples of urls in the app.